Project 2/four-card-feature/style.css

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;600&display=swap');
+
+body {
+    background-color: hsl(0, 0%, 98%);
+    color: hsl(234, 12%, 34%);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding: 0;
+    padding-top: 60px;
+}
+
+p {
+    font-size: 15px;
+}
+
+.container {
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.container .box {
+    position: relative;
+    box-shadow: 0 5px 30px hsla(229, 6%, 66%, 0.5);
+    font-family: 'Poppins';
+    font-weight: 200;
+    text-align: left;
+    height: 250px;
+    width: 314px;
+    border-radius: 10px;    
+    background-color: white;
+    margin: 30px 15px;
+    padding: 10px 30px;
+}
+
+
+.container .box>img {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    margin: 0 30px 30px 0;
+}
+
+.first-box {
+    border-top: 5px solid hsl(180, 62%, 55%);
+}
+.second-box {
+    border-top: 5px solid hsl(0, 78%, 62%);
+    margin-top: 0 !important;
```

Comment:
tak nie robimy :) jeżeli jest konieczność nadania takiego stylu, to trzeba użyć cssowego specifity :) !important to jest trytytka na kryzysowe sytuacje